GPSBabel.pro

```diff
@@ -148,7 +148,7 @@ win32 {
   }
   SOURCES += gbser_win.cc
   JEEPS += jeeps/gpsusbwin.cc
-  LIBS += "C:/Program Files/Windows Kits/8.0/Lib/win8/um/x86/setupapi.lib" "C:/Program Files/Windows Kits/8.0/Lib/win8/um/x86/hid.lib"
+  LIBS += "C:/Program Files (x86)/Windows Kits/8.1/lib/winv6.3/um/x64/setupapi.lib" "C:/Program Files (x86)/Windows Kits/8.1/lib/winv6.3/um/x64/hid.lib"
```

Comment:
I tried to use ```$(WindowsSDK_LibraryPath_x64)``` but there's a bug in visual studio preventing in from working, I believe it can be solved properly but I don't have the time to play around with it. Also I figured that no one is using a non 64bit windows system nowadays. and 8.1 replaced 8.0 so I figured this is a safer bet...